operator/v1/types_network.go

```diff
@@ -328,6 +331,19 @@ type HybridOverlayConfig struct {
 	HybridOverlayVXLANPort *uint32 `json:"hybridOverlayVXLANPort,omitempty"`
 }
 
+type OVSHardwareOffloadConfig struct {
+	// enableHardwareOffload specifies whether or not OVS hardware offload
+	// should be enabled. If unset, this property defaults to 'false' and
+	// OVS hardware offload is disabled.
+	EnableHardwareOffload bool `json:"enableHardwareOffload"`
+	// hardwareOffloadPolicy is the Traffic Control Policy (tc-policy) specified
+	// for OVS instance. This field is only relevant if EnableHardwareOffload is
+	// enabled. If unset, this property defaults to TrafficControlPolicyNone. The
+	// supported values are TrafficControlPolicyNone, TrafficControlPolicySkipSW
+	// and TrafficControlPolicySkipHW.
+	HardwareOffloadPolicy TrafficControlPolicy `json:"hardwareOffloadPolicy"`
```

Comment:
`hardwareOffloadPolicy: none` really sounds like it means "no hardware offload". I think either the field or the values need to be renamed to clarify this better. We don't necessarily need the field names to copy the underlying `tc` API.

Reply:
Yeah, the default value `none` doesn't sound like a good policy name. This is copied from ovs other_config.tc-policy values directly. I understand the policy as to control which traffic datapath shall be used when hw-offload is enabled.
when policy is defualt 'none', it won't control the datapath which in turns tries the following datapathes in order: hw datapath, ovs tc software datapath, ovs kernel software datapath. The TrafficControlPolicy give user the option to choose which datapath shall be used or skipped. does it make sense if I change the field name to OVSTrafficControlPolicy where OVS can indicates that this option is to configure openvswitch tc policy?

Reply:
Changed the field name from `HardwareOffloadPolicy` to `OVSTrafficControlPolicy`.